src/const.ts

```diff
@@ -4,9 +4,13 @@ import placeholderImage from "./content/assets/img/placeholder.svg";
 
 export const DOCS_NAVIGATION: SideNavItemConfig[] = [
     {
-        title: "Installation",
+        title: "Installation v1",
         slug: "/docs/quick-installation",
     },
+    {
+        title: "Installation V2",
```

Comment:
You are using uppercase `V2` here and lowercase `v2` and `v1` in other locations

Reply:
updated